include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -82,7 +82,7 @@ class edit_distance_unbanded_max_errors_policy :
     {
         derived_t * self = static_cast<derived_t *>(this);
 
-        max_errors = get<align_cfg::max_error>(self->config).value;
+        max_errors = -get<align_cfg::min_score>(self->config).value;
```

Comment:
Ok, this means we need to check in the edit before, that the score value is less than or equal to 0 and otherwise throw. Because that would be an invalid configuration and we explicitly say, that the min_score is only available for the edit distance. Later, we might relax this and then choose another alignment configuration if a positive score is requested, which seems quite legit to me.